analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundancyInConstructorDestructorDeclaration.CSharp12.cs

```diff
@@ -20,6 +21,11 @@ struct AStructWithFieldInitializer()              // Compliant
                 public int aField = 42;
             }
 
+            struct AStructWithFieldDoubleInitializers()       // Compliant
+            {
+                public int aField, bField = 42;
```

Comment:
Good test! 
Does/should the rule act differently if the field is not `public`?

Reply:
No it does not, I'll remove the access modifier then